core/src/main/java/org/apache/metamodel/schema/ImmutableColumn.java

```diff
@@ -71,12 +72,13 @@
      *            whether the column is a primary key or not
      */
     public ImmutableColumn(String name, ColumnType type, Table table, int columnNumber, Integer columnSize,
-            String nativeType, Boolean nullable, String remarks, boolean indexed, String quote, boolean primaryKey) {
+                           Integer decimalDigits, String nativeType, Boolean nullable, String remarks, boolean indexed, String quote, boolean primaryKey) {
```

Comment:
By adding a parameter to this constructor, the code will cause compilation errors in other project using this constructor. Can you please add an extra constructor with the signature of the original version of this constructor, which invokes this constructor with a default value for `decimalDigits`? In that manner you also don't have to change all the unit test code.

Reply:
This looks good now.